src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -411,12 +413,68 @@ public RepositoryMetadata GetRepositoryMetadata()
             return repository;
         }
 
+        public LicenseMetadata GetLicenseMetadata()
+        {
+            var ns = MetadataNode.GetDefaultNamespace().NamespaceName;
```

Comment:
namespace is a key word :)
I'd have to do `@namespace`. 

I've just removed it completely, it doesn't need it's own variable since it's not used ever again.